src/client/modules/page-not-found/containers/page-not-found.jsx

```diff
@@ -2,20 +2,23 @@ import React from 'react';
 import { Link } from 'react-router-dom';
 import Helmet from 'react-helmet';
 import { Button } from 'reactstrap';
+import Page from '../../../app/page';
 
 const pageNotFound = () => (
-  <section className="text-center mt-4 mb-4">
-    <Helmet
-      title='Apollo Starter Kit - Page not found'
-      meta={[{
-        name: 'description',
-        content: 'Apollo Starter Kit - Page not found'
-      }]}/>
-    <h2>Page not found - 404</h2>
-    <Link to="/">
-      <Button className='home-link' color="primary">Go to Homepage</Button>
-    </Link>
-  </section>
+  <Page>
+    <section className="text-center mt-4 mb-4">
```

Comment:
We can remove the `section` wrapper from all the component pages since `Page` is acting as a wrapper now.